scripts/mdf_parser.py

```diff
@@ -72,6 +77,7 @@ def write_dict_to_csv(fname, data_dict, upper_level_key_length=0):
 def main(argv):
     if (len(argv) > 0):
         read_html_data(argv[0] + '/' + argv[1])
+        fix_villages({1105130121: 'ALUE DUA MUKA O', 3506190010: 'SITIMERTO'})
```

Comment:
I think this method is quite dangerous. In case BPS rename `1105130121` and `3506190010`, the generated data will be not following BPS update. What do you think?

Reply:
https://github.com/edwardsamuel/Wilayah-Administratif-Indonesia/issues/16 is a way to check for more problems.  But I think we should not wait for all problems to be found.  They will be reported when people find them.

And we cant wait for government to fix them.  That doesnt  happen quickly.
But the fixes should be optional, so we can still use this repo's tools to obtain raw data.